invenio_oauthclient/handlers/rest.py

```diff
@@ -39,6 +40,13 @@ def response_handler_postmessage(remote, url, payload=None):
 
 def default_response_handler(remote, url, payload=None):
     """Default response handler."""
+    default_handler = current_app.config.get("OAUTHCLIENT_REST_DEFAULT_RESPONSE_HANDLER")
```

Comment:
we can reuse this https://github.com/inveniosoftware/invenio-oauthclient/blob/aee04536c2c8d1074da227026756074f823c8a81/invenio_oauthclient/utils.py#L168

Reply:
:+1: